### PR TITLE
dev-perl/Math-PlanePath: version bump 123

### PR DESCRIPTION
**Pull request by tomboy-64**
@gentoo/perl
@gentoo/proxy-maint

Package-Manager: portage-2.3.0_rc1

### CODE REVIEW

**Comment by tomboy-64** [comment]
marginal update.
tests run through just fine.

File: dev-perl/Math-PlanePath/Math-PlanePath-123.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+EAPI=6
+
+DIST_AUTHOR="KRYDE"
+DIST_VERSION="123"
+inherit perl-module
+
+DESCRIPTION="Mathematical paths through the 2-D plane"
+
+LICENSE="GPL-3"
```
Comment (gktrk) [inline dev-perl/Math-PlanePath/Math-PlanePath-123.0.0.ebuild]: This should be GPL-3+ I think

from lib/Math/PlanePath.pm:
"either version 3, or (at your option) any later version."

Comment (tomboy-64) [inline dev-perl/Math-PlanePath/Math-PlanePath-123.0.0.ebuild]: Thanks! :+1: 

Has been corrected.

**Comment by monsieurp** [comment]
@gentoo/perl

**Pull request closed without merging**